apps/mobile/src/features/account/account.tsx

```diff
@@ -44,6 +44,7 @@ export function Account({ account, walletName, balance, isLoading, isError }: Ac
                   pathname: AppRoutes.SettingsWalletConfigureAccount,
                   params: {
                     fingerprint: account.fingerprint,
+                    wallet: account.fingerprint,
```

Comment:
@edgarkhanzadian I hit some rebase issues here which I can fix but I wanted to ask about this. Is there a reason we are doubling up on this stuff?

Why do we pass `wallet: account.fingerprint` ? 

Do we need to pass `account: account.id, accountId: account.id` twice?

Reply:
OK that's great, thanks for explaining 👍